vertx-mssql-client/src/main/java/io/vertx/mssqlclient/impl/codec/DataType.java

```diff
@@ -240,7 +241,56 @@ public Object decodeValue(ByteBuf byteBuf, Metadata metadata) {
   NUMERIC(0x3F),
 
   // Variable-Length Data Types https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-tds/ce3183a6-9d89-47e8-a02f-de5a1a1303de
-  GUID(0x24),
+  GUID(0x24) {
+    @Override
+    public Metadata decodeMetadata(ByteBuf byteBuf) {
+      Metadata metadata = new Metadata();
+      metadata.length = byteBuf.readByte();
+      return metadata;
+    }
+
+    @Override
+    public JDBCType jdbcType(Metadata metadata) {
+      if (metadata.length == 16) return JDBCType.CHAR;
```

Comment:
```suggestion
      if (metadata.length == 16) return JDBCType.GUID;
```

Reply:
There is no JDBCType.GUID in https://docs.oracle.com/javase/8/docs/api/java/sql/JDBCType.html.
Any other idea?
JDBCType.OTHER?
JDBCType.OBJECT?
JDBCType.BINARY?

Reply:
Changed it to `OTHER`